test/general/__init__.py

```diff
@@ -1,12 +1,13 @@
 from argparse import Namespace
+from typing import Type, List
 
 from BaseClasses import MultiWorld
-from worlds.AutoWorld import call_all
+from worlds.AutoWorld import call_all, World
 
 gen_steps = ["generate_early", "create_regions", "create_items", "set_rules", "generate_basic", "pre_fill"]
 
 
-def setup_solo_multiworld(world_type) -> MultiWorld:
+def setup_solo_multiworld(world_type: Type[World], steps: List[str]=gen_steps) -> MultiWorld:
```

Comment:
when using typing, there should be spaces around the `=`
```suggestion
def setup_solo_multiworld(world_type: Type[World], steps: List[str] = gen_steps) -> MultiWorld:
```